apps/etterlatte-saksbehandling-ui/client/src/components/generellbehandling/GenerellbehandlingSidemeny.tsx

```diff
@@ -0,0 +1,26 @@
+import { Sidebar } from '~shared/components/Sidebar'
+import { Generellbehandling, KravpakkeUtland, Status } from '~shared/types/Generellbehandling'
+import { AttesteringMedUnderkjenning } from '~components/generellbehandling/AttesteringMedUnderkjenning'
+
+const kanAttestere = (status: Status) => {
+  return status === Status.FATTET
+}
+export const GenerellbehandlingSidemeny = (props: {
+  utlandsBehandling: Generellbehandling & { innhold: KravpakkeUtland | null }
+}) => {
+  const { utlandsBehandling } = props
+
+  const attesterbar = kanAttestere(utlandsBehandling.status)
+
+  switch (utlandsBehandling.status) {
+    case Status.OPPRETTET:
```

Comment:
Kan man bruke fall-through på disse tro?

Reply:
jepp, litt uskker på hva som burde vises for de. Men fall-through er litt scary.